README.md

```diff
@@ -171,6 +181,27 @@ client = Kubeclient::Client.new(
 )
 ```
 
+
+### Timeouts
+
+Watching never times out.
+One-off actions like `.get_*`, `.delete_*` have a timeout, can be changed:
```

Comment:
I wanted to say both that they have a finite timeout (in constrast to "Watching never times out") and that it's configurable.  Yeah, awkward.
I'll adopt "have a configurable timeout", I explain defaults below anyway.